darwin/options.py

```diff
@@ -22,6 +22,15 @@ def __init__(self):
         # AUTHENTICATE
         subparsers.add_parser("authenticate", help="Authenticate the user. ")
 
+        # SET COMPRESSION LEVEL
+        parser_compression = subparsers.add_parser("compression", help="Set compression level.")
+        parser_compression.add_argument(
+            "compression_level",
+            type=int,
+            choices=range(0, 10),
+            help="Compression level to use on uploaded data. 0 is no compression, 9 is the best.",
```

Comment:
Seems like unless this is either set a default setting, or is set required, then the user could set `--compression` without `--compression-level` and no level would be present, which would then cause a reference error to an on darwin/cli.py:L72 ?

To me, seems like it needs either:
```python
required=True
```
or
```python
default=5 #or whatever it should be!
```

Reply:
PS. Generally prefer default settings over required arguments, as this is the argparse standard, but I don't yet know how well that suits this application.

Reply:
```
> darwin compression
usage: darwin compression [-h] {0,1,2,3,4,5,6,7,8,9}
darwin compression: error: the following arguments are required: compression_level
> darwin compression 3
> darwin compression 10
usage: darwin compression [-h] {0,1,2,3,4,5,6,7,8,9}
darwin compression: error: argument compression_level: invalid choice: 10 (choose from 0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
```

Could you explain how to run it without the parameter? As this is positional argument, it can't be set to `required=True` as I understand it, and it does not make sense to set `default` value as this argument is the whole point of the command.

Reply:
No, you're right, it works without, because it's positional.  My mistake.